skopt/space.py

```diff
@@ -205,9 +198,10 @@ def bounds(self):
 
     @property
     def transformed_bounds(self):
-        if self.prior == "uniform":
+        if self.transform_ == "hypercube":
+            return (0.0, 1.0)
+        elif self.prior == "uniform":
```

Comment:
Had to look at this a few times and still not sure if mixing `self.prior` and `self.transform_` in the `if/elif` does what we want. I'd prefer splitting these into two different `if` blocks one for priors, one for transforms.